spring-xd-related-work.tex

```diff
@@ -49,15 +49,65 @@ \subsection{Spring XD and Storm}
 \item Building upon the functional stream processing model, users have the option
 to choose from Reactor\cite{reactor}, Spark Streaming or RxJava APIs to build
 complex data centric applications.
+\item As an alternative to linear streaming data pipelines, Spring XD provides 
+support for directed ayclic graphs (DAG)\cite{dag} through named channels to 
```

Comment:
One of the defining characteristics of DAG is ["that there is no way to start at some vertex v and follow a sequence of edges that eventually loops back to v again."](http://en.wikipedia.org/wiki/Directed_acyclic_graph) I don't think we can make this claim with named channels. Maybe we can claim that we can do something similar, but I would not characterize it as DAG support.